### PR TITLE
fix(schematics): do not provide `BrowserAnimationsModule` for standalone application

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/cdk/schematics/ng-add/steps/add-taiga-modules.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ function addRootTuiProvidersToBootstrapFn(
         pushToObjectArrayProperty(
             bootstrapOptions,
             `providers`,
-            `importProvidersFrom(TuiRootModule, BrowserAnimationsModule)`,
```
Comment (waterplea) [inline projects/cdk/schematics/ng-add/steps/add-taiga-modules.ts:190]: I think we need to add `provideAnimations()` if it's missing.

Comment (splincode) [reply projects/cdk/schematics/ng-add/steps/add-taiga-modules.ts:190]: fixed

File: projects/cdk/schematics/utils/push-to-array-property.ts [hunk 2]
```diff
@@ -36,5 +37,9 @@ export function pushToObjectArrayProperty(
         return;
     }
 
-    importsInitializer.addElement(initializer);
+    if (Number.isNaN(parseInt(index as unknown as string, 10))) {
```
Comment (nsbarsukov) [inline projects/cdk/schematics/utils/push-to-array-property.ts:40]: Why `as string`? number ?

File: projects/cdk/schematics/utils/push-to-array-property.ts [hunk 3]
```diff
@@ -36,5 +37,9 @@ export function pushToObjectArrayProperty(
...
+        importsInitializer.addElement(initializer);
+    } else {
+        importsInitializer.insertElement(0, initializer);
```
Comment (nsbarsukov) [inline projects/cdk/schematics/utils/push-to-array-property.ts:43]: `index` ?

File: projects/cdk/schematics/utils/push-to-array-property.ts (hunk 2 above)
Comment (splincode) [reply projects/cdk/schematics/utils/push-to-array-property.ts]: fixed

File: projects/cdk/schematics/utils/push-to-array-property.ts (hunk 3 above)
Comment (splincode) [reply projects/cdk/schematics/utils/push-to-array-property.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**